tools/submission/submission-checker.py

```diff
@@ -0,0 +1,554 @@
+]
+
+
+class Config():
+    """Select config value by mlperf version and submission type."""
+    def __init__(self, version):
+        self.base = MODEL_CONFIG.get(version)
+        self.version = version
+        self.models = self.base["models"]
+        self.seeds = self.base["seeds"]
+        self.accuracy_target = self.base["accuracy-target"]
+        self.performance_sample_count = self.base["performance-sample-count"]
+
+    def set_type(self, submission_type):
+        if submission_type is None and self.version in ["v0.5"]:
+            return
+        elif submission_type == "datacenter":
+            self.required = self.base["required-scenarios-datacenter"]
+        elif submission_type == "edge":
+            self.required = self.base["required-scenarios-edge"]
+        else:
+            raise ValueError("innvalid system type")
+
+    def get_required(self, model):
+        if self.version in ["v0.5"]:
+            return set()
+        if model not in self.required:
+            raise ValueError("model not known: " + model)
+        return set(self.required[model])
+
+    def get_accuracy_target(self, model):
+        if model not in self.accuracy_target:
+            raise ValueError("model not known: " + model)
+        return self.accuracy_target[model]
+
+    def get_performance_sample_count(self, model):
+        if model not in self.performance_sample_count:
+            raise ValueError("model not known: " + model)
+        return self.performance_sample_count[model]
+
+
+def get_args():
+    """Parse commandline."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--input", required=True, help="submission directory")
+    parser.add_argument("--version", default="v0.7", choices=list(MODEL_CONFIG.keys()), help="mlperf version")
+    parser.add_argument("--submitter", help="filter to submitter")
+    parser.add_argument("--csv", default="summary.csv", help="csv file with results")
+    args = parser.parse_args()
+    return args
+
+
+def model_map(config, model):
+    """Map models names to the official mlperf name."""
+    if model in config.models:
+        return model
+    if model in MODEL_MAPPING:
+        return  MODEL_MAPPING[model]
+    if model.startswith("mobilenet"):
+        model = "mobilenet"
+    elif model.startswith("rcnn"):
+        model = "ssd-small"
+    elif model.startswith("ssdlite") or model.startswith("ssd-inception") or model.startswith("yolo") or \
+            model.startswith("ssd-mobilenet") or model.startswith("ssd-resnet50"):
+        model = "ssd-small"
+    return model
+
+
+def list_dir(*path):
+    path = os.path.join(*path)
+    return [f for f in os.listdir(path) if os.path.isdir(os.path.join(path, f))]
+
+
+def list_files(*path):
+    path = os.path.join(*path)
+    return [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+
+
+def split_path(m):
+    return m.replace("\\", "/").split("/")
+
+
+def ignore_errors(line):
```

Comment:
Let's add a comment here explaining why these errors are acceptable

Reply:
This came from v0.5 and as far I remember have been loadgen issues at that time.
For this PR I now changed it to
```
if config.version in ["v0.5"] and ignore_errors_for_v05(line):
```
assuming this round we might not need those exceptions anymore.